radis/io/npy.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Aug 13 14:51:22 2020
+
+@author: erwan
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def npy2df(keywords, verbose=True):
+    """ Convert a dictionary of Numpy arrays storing spectroscopic information
+    to a Pandas dataframe 
+
+    Parameters    
+    ----------
+    
+    keywords: dict
+        
+        npy2df({'wav':'PATH/TO/v0.npy', 
```

Comment:
I think we could implement such a syntax (with a dictionary) rather than the current one which uses hardcoded file names. Start a dedicated Issue as a place for discussion where we make sure everything is Tested / Documented ?

Reply:
Awesome, that's exactly what I asked in the channel a few minutes back.

Reply:
I generally think that 1 PR / Issue should focus on One feature only. 
Here we're both dealing with the GPU implementation/Validation itself, and the new .npy loading format. They're interdependant and may be implemented in the same PR, but having at least 2 separate Issues will make the discussions easier (similarly: 2 slack channels [#databases](https://radis-radiation.slack.com/archives/C018C2MH5AB) and [#gpu](https://radis-radiation.slack.com/archives/C018X7XGMBP))

Reply:
Created issue #126 for further discussion